opencga-analysis/src/main/java/org/opencb/opencga/analysis/variant/VariantWalkerTool.java

```diff
@@ -33,11 +33,11 @@
 @Tool(id = VariantWalkerTool.ID, description = VariantWalkerTool.DESCRIPTION,
         scope = Tool.Scope.PROJECT, resource = Enums.Resource.VARIANT)
 public class VariantWalkerTool extends OpenCgaTool {
-    public static final String ID = "variant-walk";
+    public static final String ID = "deprecated-variant-walk";
```

Comment:
deprecated? Why not "inline" ? We will forever accept inline custom tools, but "inline variant walkers" are deprecated?

Reply:
I was told to keep it for a couple of releases more only but I'll rename it to inline